clients/wallet/src/horizon/responses.rs

```diff
@@ -319,10 +319,10 @@ pub struct HorizonBalance {
 	#[serde(deserialize_with = "de_string_to_f64")]
 	pub balance: f64,
 	#[serde(default)]
-	#[serde(deserialize_with = "de_string_to_optional_bytes")]
+	#[serde(deserialize_with = "de_str_to_optional_bytes")]
 	pub asset_code: Option<Vec<u8>>,
 	#[serde(default)]
-	#[serde(deserialize_with = "de_string_to_optional_bytes")]
+	#[serde(deserialize_with = "de_str_to_optional_bytes")]
```

Comment:
So we don't want to use `String` here? Or do we? Because I saw you added a new serde function but it's not used here.

Reply:
Maybe not. 

The issue was on deserializing special characters, which the `memo` contains now and then:
```
\"ySHX => SHX/XLM DAILY\"
```
see my commit description: https://github.com/pendulum-chain/spacewalk/pull/495/commits/e6aaa8393ac56c024394eef1b88129b4d6e39c50

Reply:
Sorry, I was a little confused because just looking at the changed lines it was not clear to me how this applies to the memo as mentioned in the PR description. But now I realized that these changed lines here are not about the memo, but other fields like e.g. [this](https://github.com/pendulum-chain/spacewalk/blob/3c68672cc6e99ddf693604a6649e33c8cf8c71ca/clients/wallet/src/horizon/responses.rs#L198) one are now using a `String` instead of the `str` which was misleading in the function name.

Makes sense then 👍  but why do we need to use `str` for asset code and issuer? Or is it just because we know that there cannot be any weird characters?

Reply:
I did not want to change how the other fields are deserialized. 
Maybe it's even best that we prevent such special characters in these fields.

Reply:
see https://github.com/pendulum-chain/spacewalk/pull/495/commits/c9fe6a4a5129776f258300b32ab57f3b53424e3e

Reply:
Yes, makes sense 👍